docs/develprocess.md

```diff
@@ -108,6 +108,61 @@ Our policy is to upstream everything if possible. However, there are some except
 ### How to help at development
 It all depends on your skills and areas of interest so it's hard to tell specifically in advance. Having a look at the open github issues and pick one (<https://github.com/xcp-ng/xcp/issues>) could definitely help. Else, maybe there's a specific topic that you would want to help improve. Even if you don't know where to start, just come and talk with us (see "Where discussion happens" above).
 
+## Tags, maintenance branches in our code repositories
+
+Objectives of the branch and tag naming conventions:
+* always know how to name tags and maintenance branches depending on the situation
+* easily identify maintenance branches for a given release of XCP-ng, based on their name
+* know what branch to develop the next version on
+* for our tags and branches that have been developed from upstream branches or tags, document the upstream branch names through our branch and tag naming
+
+First question to ask ourselves: **who is the upstream for the software**?
+
+### 1. We are upstream
+
+Example: `xcp-emu-manager`
+* Tags: `vMAJOR.MINOR.PATCH` (`v1.1.2`, `v1.2.0`...)
+* Maintenance branch if needed: `VERSION-XCPNGVERSION}`.
```

Comment:
```suggestion
* Maintenance branch if needed: `{VERSION-XCPNGVERSION}`.
```

Reply:
or perhaps no `{}` at all :)

Reply:
Indeed, I had started adding {} everywhere then I decided against it :)

Reply:
Done